the_pingler.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/zsh
+
+set -o shwordsplit
+
+API="http://localhost:9001/management/healthcheck"
+THRALL="http://localhost:9002/"
+IMAGE_LOADER="http://localhost:9003/management/healthcheck"
+FTP_WATCHER="http://localhost:9004/"
+KAHUNA="http://localhost:9005/management/healthcheck"
+CROPPER="http://localhost:9006/management/healthcheck"
+METADATA="http://localhost:9007/management/healthcheck"
+USAGE="https://localhost:9009/"
```

Comment:
It's in a different place for `ftp-watcher`: `/health-check`. `thrall` reports failed health-check sometime because of the message frequency check.